tools/scripts/sim-update-powerfail-resume-nobackup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+V=`./wolfboot.elf update_trigger get_version 2>/dev/null`
+if [ "x$V" != "x1" ]; then
+    echo "Failed first boot with update_trigger"
+    exit 1
+fi
+
+./wolfboot.elf powerfail 0 get_version 2>/dev/null
+./wolfboot.elf powerfail 15000 get_version 2>/dev/null
+./wolfboot.elf powerfail 18000 get_version 2>/dev/null
+./wolfboot.elf powerfail 1a000 get_version 2>/dev/null
+# fail on the last sector to stop the encrypt key save and state update
+./wolfboot.elf powerfail 3e000 get_version 2>/dev/null
+# may not trigger on non NVM_FLASH_WRITEONCE
```

Comment:
does `NVM_FLASH_WRITEONCE` get used with `DISABLE_BACKUP`? Also missing combo cases for things like `HOME_FLAGS`, not sure if it's relevant for this flag but otherwise looks good.